src/ducks/settings/DelayedDebitAlert.jsx

```diff
@@ -95,34 +100,47 @@ class DumbDelayedDebitAlert extends React.Component {
           enabled={enabled}
           name="delayedDebit"
         />
-        <Stack>
-          {accountsAssociations.map((association, index) => (
-            <AccountsAssociationSelect
-              creditCardOptions={creditCardOptions}
-              checkingsOptions={checkingsOptions}
-              association={association}
-              enabled={enabled}
-              key={association.creditCardAccount + association.checkingsAccount}
-              onChange={newAssociation =>
-                onAccountsAssociationChange(index, newAssociation)
-              }
-            />
-          ))}
-          {accountsAssociations.length === 0 ? (
-            <AccountsAssociationSelect
-              creditCardOptions={creditCardOptions}
-              checkingsOptions={checkingsOptions}
-              association={{
-                creditCardAccount: null,
-                checkingsAccount: null
-              }}
-              enabled={enabled}
-              onChange={newAssociation =>
-                onAccountsAssociationChange(0, newAssociation)
-              }
-            />
-          ) : null}
-        </Stack>
+        {hasNoAccount && (
+          <Text>{t('Notifications.delayed_debit.noAccount')}</Text>
+        )}
+        {!hasNoAccount && hasNoCreditCard && (
+          <Text>{t('Notifications.delayed_debit.noCreditCard')}</Text>
+        )}
+        {!hasNoAccount && hasNoCheckings && (
+          <Text>{t('Notifications.delayed_debit.noCheckings')}</Text>
+        )}
+        {!hasNoAccount && !hasNoCreditCard && !hasNoCheckings && (
```

Comment:
Nit: Having a boolean variable as a negative is a bit confusing IMHO. I prefer hasAccount and !hasAccount.